sdk/storage/azure-storage-file-share/out/azure/storage/fileshare/_file_client.pyi

```diff
@@ -30,9 +31,10 @@ class ShareFileClient(StorageAccountHostsMixin):
     def from_file_url(cls: Any, file_url: str, snapshot: Optional[Union[str, Dict[str, Any]]]=..., credential: Optional[Any]=..., **kwargs: Any) -> ShareFileClient: ...
     @classmethod
     def from_connection_string(cls: Any, conn_str: str, share_name: str, file_path: str, snapshot: Optional[Union[str, Dict[str, Any]]]=..., credential: Optional[Any]=..., **kwargs: Any) -> ShareFileClient: ...
+    def acquire_lease(self, lease_id: Optional[str]=..., **kwargs: Any) -> BlobLeaseClient: ...
     def create_file(self, size: int, file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., permission_key: Optional[str]=..., **kwargs: Any) -> Dict[str, Any]: ...
     def upload_file(self, data: Any, length: Optional[int]=..., file_attributes: Union[str, NTFSAttributes]=..., file_creation_time: Union[str, datetime]=..., file_last_write_time: Union[str, datetime]=..., file_permission: Optional[str]=..., permission_key: Optional[str]=..., **kwargs: Any) -> Dict[str, Any]: ...
-    def start_copy_from_url(self, source_url: str, **kwargs: Any) -> Any: ...
+    def start_copy_from_url(self, source_url: str, file_permission_copy_mode: Optional[str]=..., file_permission: Optional[str]=..., file_permission_key: Optional[str]=..., copy_file_smb_info: Any=..., **kwargs: Any) -> Any: ...
```

Comment:
The new parameters should be `**kwargs`

Reply:
We would like more extensive changes to the parameters for this operation - see conversation in email thread.